scanners/ctlog_scanner.py

```diff
@@ -0,0 +1,46 @@
+import os
+import certstream
+import logging
+import json
+
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.INFO)
+
+
+class CTLogScanner():
```

Comment:
We should have unit-tests included for this, so it's more clear as to how our interfaces are to work.  I  may take a stab at doing that and propose as a PR to this PR.